client/src/components/Banner/Banner.css

```diff
@@ -0,0 +1,28 @@
+.Banner{
+  padding: 0;
+  margin: 0 auto;
+  width: 100%;
+  width: 100vw;
+  height: 231px;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-align: center;
+  font-family: var(--default-font);  
+  font-size: 48px;
+  font-style: normal;
+  font-weight: normal;
+  color: #FAFAFA;
+  line-height: 72px;
+}
+
+.Banner__header3{
+  margin: 10px auto;
+}
+
+.Banner .InputTextField__container--search{
+  margin: 0 auto;
+}
```

Comment:
We never want to affect the CSS inside another component. If you need to change the look of another component, then you need to update that component